// ANCHOR: all
// ANCHOR: command
$ forge test --zksync --allow-failure
// ANCHOR_END: command
// ANCHOR: output
Compiling 1 files with Solc 0.8.10
Solc 0.8.10 finished in 941.22ms
Compiler run successful with warnings:
Warning (9302): Return value of low-level calls not used.
  --> test/Safe.t.sol:11:9:
   |
11 |         payable(msg.sender).call{value: address(this).balance}("");
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Warning (9302): Return value of low-level calls not used.
  --> test/Safe.t.sol:30:9:
   |
30 |         payable(address(safe)).call{value: amount}("");
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^


Compiling 1 files with zksolc and solc 0.8.10
zksolc and solc 0.8.10 finished in 5.14s
Compiler run successful with warnings:
Warning (9302)
Warning: Return value of low-level calls not used.
  --> test/Safe.t.sol:11:9:
   |
11 |         payable(msg.sender).call{value: address(this).balance}("");
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Warning (9302)
Warning: Return value of low-level calls not used.
  --> test/Safe.t.sol:30:9:
   |
30 |         payable(address(safe)).call{value: amount}("");
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

2025-01-09T13:58:44.187764Z ERROR foundry_zksync_core::vm::tracers::cheatcode: call may fail or behave unexpectedly due to empty code target=0x7fa9385be102ac3eac297483dd6233d62b3e1496 calldata=""
2025-01-09T13:58:44.200605Z ERROR foundry_zksync_core::vm::tracers::cheatcode: call may fail or behave unexpectedly due to empty code target=0x7fa9385be102ac3eac297483dd6233d62b3e1496 calldata=""
2025-01-09T13:58:44.206499Z ERROR foundry_zksync_core::vm::inspect: tx execution halted: Account validation error: Error function_selector = 0x4e487b71, data = 0x4e487b710000000000000000000000000000000000000000000000000000000000000011
2025-01-09T13:58:44.212336Z ERROR foundry_zksync_core::vm::tracers::cheatcode: call may fail or behave unexpectedly due to empty code target=0x7fa9385be102ac3eac297483dd6233d62b3e1496 calldata=""

Ran 1 test for test/Safe.t.sol:SafeTest
[FAIL: panic: arithmetic underflow or overflow (0x11); counterexample: calldata=0x29facca7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffd args=[115792089237316195423570985008687907853269984665640564039457584007913129639933 [1.157e77]]] testFuzz_Withdraw(uint256) (runs: 2, Î¼: 254285, ~: 254285)
Suite result: FAILED. 0 passed; 1 failed; 0 skipped; finished in 47.57ms (37.50ms CPU time)

Ran 1 test suite in 48.20ms (47.57ms CPU time): 0 tests passed, 1 failed, 0 skipped (1 total tests)
// ANCHOR_END: output
// ANCHOR_END: all
